core/src/main/kotlin/org/evomaster/core/search/gene/SqlUUIDGene.kt

```diff
@@ -27,19 +32,21 @@ class UUIDGene(
     }
 
     override fun getValueAsRawString(): String {
-        return UUID(mostSigBits.value, leastSigBits.value).toString()
+        return "{%s}".format(getValueAsUUID())
     }
```

Comment:
please comment on `"{%s}".format`

Reply:
I checked that postgresql allows inserting uuid values without any enclosing braces {...}